src/config/define-config.ts

```diff
@@ -0,0 +1,69 @@
+import * as v from 'valibot'
+
+import type { JestConfigWithTsJest } from '../types'
+
+/**
+ * Future config options for experimental features
+ */
+export interface FutureConfig {
```

Comment:
```suggestion
export interface FutureConfig {
    validateOptions: boolean
}
```

and adopt this suggestion for this file.

When users set `validateOptions: true`, we should call the function from `src/experimental/option-validator` in `futureConfigSchema` to validate the options